packages/wrangler/src/metrics/metrics-dispatcher.ts

```diff
@@ -111,6 +111,7 @@ export function getMetricsDispatcher(options: MetricsConfigOptions) {
 					isPagesCI: isPagesCI(),
 					isWorkersCI: isWorkersCI(),
 					isInteractive: isInteractive(),
+					hasAssets: options.hasAssets ?? false,
```

Comment:
this might be a tad misleading, since if `options.hasAssets` is `undefined`, we'll default to `false`, which might not be entirely true. However, the only use case I can see this happening is for code paths that call the old `metrics.sendMetricsEvent()`. The new metricsDispatcher will handle this correctly